builtin/logical/pki/acme_state_test.go

```diff
@@ -29,6 +29,7 @@ func TestAcmeNonces(t *testing.T) {
 		nonce, _, err = a.GetNonce()
 		require.NoError(t, err)
 		require.NotEmpty(t, nonce)
+		nonces = append(nonces, nonce)
```

Comment:
I am not 100% this is what was intended here, but it's my best guess. Otherwise `nonces` is always empty and the code below is a no-op.

@cipherboy in case you remember what you wanted to do here 2 years ago 😄

Reply:
Ah, yes, good catch.